pytransform3d/urdf.py

```diff
@@ -102,17 +103,22 @@ def set_joint(self, joint_name, value):
         from_frame, to_frame, child2parent, axis, limits, joint_type = \
             self._joints[joint_name]
         # this is way faster than np.clip:
-        value = min(max(value, limits[0]), limits[1])
+        value_lim = min(max(value, limits[0]), limits[1])
+        if value_lim != value:
+            warnings.warn(f"Value for {joint_name} is capped by limits")
```

Comment:
This changes existing behavior, which I would like to avoid. Values for other joint types were just clipped silently before. I think that's generally accepted behavior. If it's not documented in the function's docstring, we could think about documenting this behavior.

Reply:
OK, understood, I've undone these changes.